### PR TITLE
Add issue template for kind/cleanup

### LINKED ISSUE #3392: Need an issue template for /kind cleanup
/kind cleanup

**Describe the solution you'd like**
Currently, there is no PR template for issues with the `cleanup` label. This has resulted in many cleanup issues being mislabeled as bugs or features. In addition, this would help improve process for creating cleanup issues.

**Environment:**

- cluster-api-provider-azure version: 
- Kubernetes version: (use `kubectl version`): 
- OS (e.g. from `/etc/os-release`):

### PR DESCRIPTION
**Pull request by willie-yao**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR adds an issue template for `kind/cleanup`.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #3392 

**Special notes for your reviewer**:

- [ ] cherry-pick candidate

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

File: .github/ISSUE_TEMPLATE/cleanup_request.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: 'Cleanup request'
+about: Propose a cleanup task for this project.
+
+---
+
+/kind cleanup
+
+**What needs cleanup:**
+[A clear and concise description of what needs to be cleaned up.]
```
Comment (Jont828) [inline .github/ISSUE_TEMPLATE/cleanup_request.md:10]: WYDT about adding something to the template to include a way to specify which parts to clean up, i.e. line numbers, files, etc?

Comment (willie-yao) [reply .github/ISSUE_TEMPLATE/cleanup_request.md:10]: Just added a checklist for this! Let me know what you think.

**Review by mboersma — COMMENTED** [review]
/lgtm

**Pull request merged**